### PR TITLE
New features

### PR DESCRIPTION
**Pull request by gandarez**
* consider portable mode when calling getUserHomeDir
* add new commands
* use string interpolation

### CODE REVIEW

File: src/wakatime.ts [hunk 1]
```diff
@@ -155,6 +154,26 @@ export class WakaTime {
         });
     }
 
+    public getConfigFilePath(): void {
```
Comment (alanhamlett) [inline src/wakatime.ts]: Let's name this `openConfigFile()` since it's opening the file.

File: src/wakatime.ts [hunk 2]
```diff
@@ -155,6 +154,26 @@ export class WakaTime {
...
+        let path = this.options.getConfigFile();
+        if (path) {
+            let uri = vscode.Uri.file(path);            
+            vscode.window.showTextDocument(uri).then(doc => {
+                doc.viewColumn = vscode.ViewColumn.Beside;
+            });
+        }
+    }
+
+    public getLogFilePath(): void {
```
Comment (alanhamlett) [inline src/wakatime.ts]: Same here.

File: src/wakatime.ts (hunk 1 above)
Comment (gandarez) [reply src/wakatime.ts]: Initially it supposed to give the path but I changed to open file. thanks for pointing that.

**Comment by gandarez** [comment]
@alanhamlett done!

**Comment by alanhamlett** [comment]
Thank you!

**Comment by jgmuchiri** [comment]
@gandarez @alanhamlett  Thank you guys for addressing this!

**Pull request merged**